src/compositor/mod.rs

```diff
@@ -151,7 +151,7 @@ impl Compositor {
     }
 }
 
-#[derive(Debug, Copy, Clone)]
+#[derive(Copy, Clone)]
```

Comment:
Why no `Debug`?

Reply:
Because `[tracking::TrackedDevicePose; 64]` does not implement Debug - not sure why this has not been an issue before.

Reply:
Ah, it used to be 16.